airbyte-ci/connectors/base_images/base_images/templates/README.md.j2

```diff
@@ -79,6 +79,9 @@ poetry run mypy base_images --check-untyped-defs
 
 ## CHANGELOG
 
+### 1.6.0
```

Comment:
Funny, but it seems that the changelog for the package is in a _template_ file. Pretty insane tbh.